docker-compose.yml

```diff
@@ -4,10 +4,15 @@ services:
   benchmark-ann:
     build: 
       context: ./
-      dockerfile: Dockerfile-ann
+      dockerfile: Dockerfile-ann1b
```

Comment:
I think we should duplicate the docker-compose file to have one for the small benchmarks and one for the 1b benchmarks, because we are also referencing this repo in the publishing of the benchmarks, which would use the other Dockerfile.

Can you create a second docker-compose and add your changes there?

Reply:
I can rename this one and the merge is going to have both of them.

Reply:
I can rename this one and the merge is going to have both of them. Does this work?

Reply:
Perfect.